crates/loader/src/assets.rs

```diff
@@ -90,6 +90,19 @@ pub fn file_sha256_digest_string(path: impl AsRef<Path>) -> std::io::Result<Stri
     Ok(digest_string)
 }
 
+/// Return an error if a path is not under a given directory.
```

Comment:
Apologies for missing this before but I think this doc string is incorrect?

Reply:
Hi @itowlson, no worries, it's my fault that I didn't check it carefully. Thanks for catching it!